cli/main.go

```diff
@@ -58,7 +58,8 @@ func Run(host string, port int) {
 			Fn: func(buf *prompt.Buffer) {
 				if dicedbClient.subscribed {
 					fmt.Println("Exiting watch mode.")
```

Comment:
@arpitbbhayani I think this message log also should be moved to handleWatchModeExit method.